server/player/handler.go

```diff
@@ -22,6 +22,8 @@ type Handler interface {
 	HandleMove(ctx *event.Context, newPos mgl64.Vec3, newYaw, newPitch float64)
 	// HandleTeleport handles the teleportation of a player. ctx.Cancel() may be called to cancel it.
 	HandleTeleport(ctx *event.Context, pos mgl64.Vec3)
+	// HandleChangeWorld handles when the player is added to a new world.
```

Comment:
Might be a good idea to document that these worlds may be nil when added to a world for the first time or removed at the end.